Assets/Scripts/Projectile.cs

```diff
@@ -5,6 +5,12 @@
 public class Projectile : MonoBehaviour
 {
     public GameObject hitEffect; //put some explosion or some shit here.
+    public float timeToLive = 2.0f;
+
+    private void Start()
+    {
+        Destroy(gameObject, 2.0f);
```

Comment:
why not use that cool variable you declared up there.

Reply:
Should be done

Reply:
frick off